renderer/views/torrent-list.js

```diff
@@ -114,3 +118,45 @@ function renderTorrentButtons (torrentSummary, dispatch) {
     </div>
   `
 }
+
+// Show files, per-file download status and play buttons, and so on
+function renderTorrentDetails (torrent, torrentSummary) {
+  var filesElement
+  if (!torrent || !torrent.files) {
+    // We don't know what files this torrent contains
+    var message = torrent
+      ? 'Downloading torrent data using magnet link...'
+      : 'Failed to download torrent data from magnet link. Click the download button to try again...'
+    filesElement = hx`<div class='files warning'>${message}</div>`
+  } else {
+    // We do know the files. List them and show download stats for each one
```

Comment:
Ideally this should be in the webtorrent library

Currently the webtorrent file object just has a property, `done`, that says whether all pieces are loaded. Maybe we could add `numPieces` and `numPiecesLoaded` to show progress.

This would also be more efficient, since webtorrent can increment `numPieces` every time a new piece comes in, instead of us iterate thru all pieces in the file to see which ones are there